auth/services/x509/uzi_validator.go

```diff
@@ -173,13 +174,15 @@ func NewUziValidator(env UziEnv, contractTemplates *contract.TemplateStore, crlV
 	}
 
 	if crlValidator == nil {
-		crlValidator = crl.NewValidator(append(roots[:], intermediates...))
+		crlValidator = crl.New(append(roots[:], intermediates...))
 	}
 
 	validator = &UziValidator{
 		validator:         NewJwtX509Validator(roots, intermediates, validUziSigningAlgs(), crlValidator),
 		contractTemplates: contractTemplates,
 	}
+	// TODO: context is never cancelled
+	crlValidator.Start(context.TODO())
```

Comment:
What will we replace this with?

Reply:
see the [make uzi crl validator cancellable](https://github.com/nuts-foundation/nuts-node/pull/2011/commits/9e501ffb2be629c4572d56f679a8d8acb887db23) commit. 
Big changes for little impact. Although I do think that this is a better approach.